include/nuttx/sdio.h

```diff
@@ -477,6 +477,7 @@
 #define SDIO_CAPS_DMABEFOREWRITE  0x04 /* Bit 2=1: Executes DMA before write command */
 #define SDIO_CAPS_4BIT            0x08 /* Bit 3=1: Supports 4 bit operation */
 #define SDIO_CAPS_8BIT            0x10 /* Bit 4=1: Supports 8 bit operation */
+#define SDIO_CAPS_4BIT_ONLY       0x20 /* Bit 5=1: Supports 4-bit only operation */
```

Comment:
why not remove SDIO_CAPS_8BIT from cap?

Reply:
The issue is that the peripheral does not support 1bit mode, so 4 bit mode must be selected _before_ checking the SCR register.

Reply:
@AndrewD  Is correct. This peripheral _only_ supports 4 bit mode.
I.e. It doesnt support 1bit _or_ 8bit modes.

We therefore must have a mechanism to force the mmcsd driver to put the card into 4 bit mode without checking the capabilities of the card obtained from the SCR register.

Reading the SCR register becomes redundant but has been left in place for diagnostic purposes - If DEBUG_FS_INFO is enabled.